src/app/main/components/header/header.component.scss

```diff
@@ -4,11 +4,11 @@
   display: flex;
```

Comment:
it's general comment not for this file 

main components should be near main routing module . delete main-page folder and put the files together
html, spec, ts, scss, module, routing module
it will be easier to read and understand nesting logic

Reply:
done. Thanks